src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -85,8 +98,18 @@ const EditShiftHours = ({
     );
 
     const endTime = formData[`${inputName}-finish-time`] || null;
+    var actualEndDate = formatDate(timecardDate);
```

Comment:
we should avoid using `var `where not necessary as it's function scoped. 
might need to use `let `as it's reassigned a value on ln 104